pymatviz/powerups.py

```diff
@@ -196,12 +196,10 @@ def annotate_metrics(
         "MSE": lambda x, y: ((x - y) ** 2).mean(),
         "MAPE": mape,
         "R2": r2_score,
-        # TODO: check this for correctness
```

Comment:
I think the formula for adjusted R2 is correct, as per [the reference](https://www.statisticshowto.com/probability-and-statistics/statistics-definitions/adjusted-r2/) here:
![image](https://github.com/janosh/pymatviz/assets/80093591/189413c0-d18d-4091-84f2-5d26cdd22325)

In our case the `k` (the number of independent regressors, i.e. the number of [variables](https://www.statisticshowto.com/probability-and-statistics/types-of-variables/) in your model) is 1 (single y only)